Add TokenEndPoint and AuthorizationEndpoint explanation (#4444)
umbraco/UmbracoDocs: Reference/Security/Authenticate-with-Active-Directory/index.md

    ```

    :::note

>     Ensure to replace **{your_client_id}** and **{your_client_secret}** in the code with the values from the Azure AD tenant. If Azure AD is configured to use only accounts in the organizational directory only (single tenant), you have to specify the Token and AuthorizationEndpoint as well.

Reviewer: ```suggestion
    Ensure to replace **{your_client_id}** and **{your_client_secret}** in the code with the values from the Azure AD tenant. If Azure AD is configured to use accounts in the organizational directory only (single tenant), you also have to specify the Token and AuthorizationEndpoint.
```
